collect_app/src/main/java/org/odk/collect/android/dao/helpers/FormsDaoHelper.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright (C) 2018 Shobhit Agarwal
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package org.odk.collect.android.dao.helpers;
+
+import android.database.Cursor;
+
+import org.odk.collect.android.dao.FormsDao;
+import org.odk.collect.android.provider.FormsProviderAPI;
+
+public final class FormsDaoHelper {
+
+    private FormsDaoHelper() {
+
+    }
+
+    public static int getFormsCount(String selection, String[] selectionArgs) {
```

Comment:
Is the -1 meant to indicate an error?

Reply:
Might be good to document that. Or fail with a RuntimeException if it should never happen and the callers don’t deal with it?